arm: Export needed symbol for ZERO_PAGE usage in modules.

From: "Aneesh Kumar K.V" <aneesh.kumar@linux.vnet.ibm.com>

ext4 use ZERO_PAGE(0) to zero out blocks. We need to export
different symbols in different arch for the usage of ZERO_PAGE
in modules.

Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>

---
 arch/arm/mm/mmu.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.25-rc6/arch/arm/mm/mmu.c
===================================================================
--- linux-2.6.25-rc6.orig/arch/arm/mm/mmu.c	2008-03-16 16:32:14.000000000 -0700
+++ linux-2.6.25-rc6/arch/arm/mm/mmu.c	2008-03-25 14:47:40.000000000 -0700
@@ -35,6 +35,7 @@ extern pgd_t swapper_pg_dir[PTRS_PER_PGD
  * zero-initialized data and COW.
  */
 struct page *empty_zero_page;
+EXPORT_SYMBOL(empty_zero_page);
 
 /*
  * The pmd table for the upper-most set of pages.
